src/sentry/monkey/__init__.py

```diff
@@ -32,6 +32,58 @@ def safe_httprequest_repr(self):
     HttpRequest.__repr__ = safe_httprequest_repr
 
 
+def patch_django_runserver():
+    # This patch from django@934acf1126995f6e6ccba5947ec8f7561633c27f is needed
+    # to fix symbolicator tests on Django 2.0. Otherwise, symbolicator will attempt
+    # to GET DebugFilesEndpoint and fail when django liveserver closes conns prematurely.
+    # Can be removed in Django >= 2.1.4.
+
+    import django.core.servers.basehttp
+
+    def _cleanup_headers(self):
+        super(django.core.servers.basehttp.ServerHandler, self).cleanup_headers()
```

Comment:
TIL `super()` can't be used because interpreter will fill in the details depending on context when compiling to bytecode :)